src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/Testing/RunTestsHandler.cs

```diff
@@ -52,8 +52,8 @@ public async Task<RunTestsPartialResult[]> HandleRequestAsync(RunTestsParams req
             TraceLevel = GetTraceLevel(serverConfiguration),
             EnvironmentVariables = new()
             {
-                // Reset dotnet root to the user's original configuration so that vs test console can find the right runtimes.
-                { DotnetCliHelper.DotnetRootEnvVar, DotnetCliHelper.GetUserDotnetRoot() },
+                // Reset dotnet root so that vs test console can find the right runtimes.
+                { DotnetCliHelper.DotnetRootEnvVar, string.Empty },
```

Comment:
Could this be removed? Or vstest is expecting an environment variable with an empty value?

Reply:
By default DOTNET_ROOT will be the server's runtime, so we have to reset the value to empty before we start tests (otherwise they might try and run on the DOTNET_ROOT) which we don't want.